templates/KnockoutSpa/ClientApp/boot.ts

```diff
@@ -1,8 +1,8 @@
 import './css/site.css';
 import * as ko from 'knockout';
-import { createHistory } from 'history';
 import './webpack-component-loader';
 import AppRootComponent from './components/app-root/app-root';
+var createHistory = require('history').createBrowserHistory
```

Comment:
It seems to be connected to tsconfig.json "target": "es5". Switching to es6 results in an error with crossroads.